app/models/job.rb

```diff
@@ -6,7 +6,8 @@ class Job < ActiveRecord::Base
   has_one :deploy
 
   # Used by status_panel
-  alias_attribute :start_time, :created_at
+  alias_attribute :started_at, :created_at
+  alias_attribute :finished_at, :updated_at
```

Comment:
Sure you don't want to add a `finished_at` column to Job?  'Cause right now, if you called `job.finished_at` while the Job is still running, you'll get a non-nil value back.

Should be relatively easy to add a column and do a backfill.